staging/src/k8s.io/code-generator/_examples/apiserver/informers/internalversion/example2/interface.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // Code generated by informer-gen. DO NOT EDIT.
 
-package example
+package example2
```

Comment:
I think it is even better after the changes, because in `go` by convention package name is the last element of the package's path.